test/test_coordinates.py

```diff
@@ -24,6 +24,10 @@ def test_basic(self):
         self.assertEqual(coord.first, coord.x)
         self.assertEqual(coord.second, coord.y)
         self.assertEqual(coord, coord.copy())
+        self.assertEqual(str(coord), "(1, 2)")
+        self.assertEqual(hash(coord), 3713081631934410656)
```

Comment:
That makes sense.  __hash__ needs to do work without error, but I imagine we're happy as long as it returns an int.  I can update that in a separate commit.